### PR TITLE
Add BaxNetzwerk Server-Media Data

### PR DESCRIPTION
**Pull request by Ente3**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/baxnetzwerk/Manifest.json
Comment (maaaathis) [inline minecraft_servers/baxnetzwerk/Manifest.json:1]: file has to be `manifest.json`

File: minecraft_servers/baxnetzwerk/Manifest.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "%.BaxNetzwerk.de",
+      "%.play-bax.de"
+    ],
+    "supported_languages": [
+      "de",
+      "en"
+    ],
+    "social": {
+      "web": "https://BaxNetzwerk.de",
+      "web_support": "https://BaxNetzwerk.de/dev",
```
Comment (maaaathis) [inline minecraft_servers/baxnetzwerk/Manifest.json:15]: `web_support` cannot be a development progress bar

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by maaaathis** [comment]
Server is currently not open to public 
image

**Comment by DoJapHD** [comment]
PR inactive

**Pull request closed without merging**